sdk/zksend/src/channel/index.ts

```diff
@@ -120,4 +120,11 @@ export class ZkSendHost {
 			this.#request.origin,
 		);
 	}
+
+	close(payload?: ZkSendResponsePayload) {
```

Comment:
FWIW idk how important this is to have here vs just doing it in the app, but fine having it